workbench/electron-builder-config.js

```diff
@@ -42,12 +42,12 @@ const config = {
   artifactName: ARTIFACT_NAME,
   mac: {
     category: 'public.app-category.business',
-    icon: 'resources/InVEST-2.ico',
+    icon: 'resources/InVEST-2-574x574.ico',
```

Comment:
Woah, 574 is a fun dimension size. Was 512x512 too small as well or was there a particular reason for going 574x574?

Reply:
574x574 was the size of the original image before this PR, so I just stuck with that.